examples/nft/scripts/transfer/common.js

```diff
@@ -1,9 +1,38 @@
-const { executeTransaction } = require("algob");
+const { executeTransaction, readGlobalStateSSC, readLocalStateSSC } = require("algob");
 
 exports.executeTransaction = async function (deployer, txnParams) {
     try {
         await executeTransaction(deployer, txnParams);
     } catch (e) {
         console.error('Transaction Failed', e.response.error);
     }
-}
\ No newline at end of file
+}
+
+exports.printGlobalNFT = async function (deployer, creator, appId) {
+    try {
+        const globalState = await readGlobalStateSSC(deployer, creator, appId);
+        for(const g of globalState){
+            const key = Buffer.from(g.key, 'base64').toString();
+            if(key === 'total'){
+                console.log('Global NFT Count:', g.value.uint);
+            }
+        }
+    } catch (e) {
+        console.error('Error Occurred', e);
+    }
+}
+
+exports.printLocalNFT = async function (deployer, account, appId) {
+    try {
+        const localState = await readLocalStateSSC(deployer, account, appId);
+        let count = 0;
+
+        // length/2 because each nft is a pair of {id: ref_data, id_h: ref_hash}
+        if(localState !== undefined){ 
+            count = localState.length/2; 
```

Comment:
Or we can directly make count = 1,
what happens when this NFT is transferred to another user ? will this localState will become undefined for this user ?

Reply:
localState undefined means that there is nothing present in the local `ledger` (`accountInfo['apps-local-state']`). `goal` returns null. But if we are using sdk then it means we are trying to access value from an empty state (i.e if no nft is present, local state is empty), so we get undefined. 
So, if account A holds **one** NFT, and that nft is transferred to another account, then yes, localState becomes `undefined` (explained above). Let me know if that makes sense.

Reply:
Okay, that means we can make count one if local state is defined and count zero if it is undefined.

Reply:
`count = 0` should be the default. If localState is `defined`, i am updating the count (as this means there are NFT's present in the `ledger`). Am i missing something here?

Reply:
No, you are right, I thought  `localState.length/2; ` will always be one, but one application contract can have multiple NFT.
How about updating this to show NFT name and balance ? instead of showing total number of NFTs.

Reply:
We can, but this function's purpose is to display `balance` before and after the transaction. If there are multiple NFT's (say 8), before the txn we display 8 nft-names for acc_A, and after the txn we display 7. Isn't it a but cleaner the current way (just showing count)?
Also, we already have functions to print local and global state data: `printLocalStateSSC`, `printGlobalStateSSC`. We can use these functions directly to display state data. We may need to refactor to display `base64-decoded` data.